src/h.js

```diff
@@ -28,6 +29,10 @@ export function h(nodeName, attributes) {
 		if ((child = stack.pop()) instanceof Array) {
 			for (i=child.length; i--; ) stack.push(child[i]);
 		}
+		else if (typeof ATTR_KEY === 'symbol' && child[Symbol.iterator]) {
```

Comment:
`typeof ATTR_KEY === 'symbol'` avoids a type error if `Symbol` is not natively provided but presumably it also means that this won't work if Symbol is polyfilled, since I presume `typeof` is not polyfilled. I think that's an OK limitation for an initial merge, but something to be aware of.